systemtest/src/main/resources/StrimziUpgradeST.json

```diff
@@ -63,7 +63,7 @@
       "logMessageVersion": "2.2"
     },
     "client": {
-      "beforeKafkaUpdate": "strimzi/test-client:0.11.4-kafka-2.1.0",
+      "beforeKafkaUpdate": "strimzi/test-client:0.12.1-kafka-2.1.0",
```

Comment:
Could you share some more details to what these changes mean and why are they needed? I assume the part below adds upgrade to 0.15.0 to the upgrade pipeline between 0.14.0 and HEAD. But this is not clear to me - is it just fixing some bugs? Or removing tests for 0.11.4?

Reply:
No, this field is for tests client image which we use. Client images fields were there for some time, but we actually didn't use them until this time. Between versions `0.11x` and `0.12.x` I did some changes to this test-client image (changed startup script inside the image) and this is the simplest way how to use our clients mechanism for tests with these strimzi versions. In case we will leave there 0.11.x test client, test will always fail with wrong parameters for consumer/producer start-up script.